cmake/os_flags.cmake

```diff
@@ -217,7 +217,7 @@ if(WIN32)
     # Compiler specific flags
 
     ie_add_compiler_flags(/bigobj)
-    ie_add_compiler_flags(/MP)
+    ie_add_compiler_flags(/MP /FS)
```

Comment:
from https://docs.microsoft.com/en-us/cpp/build/reference/fs-force-synchronous-pdb-writes?view=msvc-160:

```
The /MP option enables /FS by default.
```

Reply:
Yes, and in the same time it doesn't work as we can see in build results for commit https://github.com/openvinotoolkit/openvino/commit/d39f54df03760950bccfd6029db8461f3567266d